### ISSUE TITLE
[quete] [46] [Grand livre de Tanaris]

### ISSUE THREAD

**Issue by Ferro3** [way-of-elendil/3.3.5#1357]
[46] [Grand livre de Tanaris] https://woehead.way-of-elendil.fr/?quest=4450
le Livre de Bonnacier https://woehead.way-of-elendil.fr/?item=11727
à coté des toilettes n'est pas lootable (cache vérifié)

l'objet doit être demandé au pnj dans les toilettes, il manque la bulle pour faire la demande, le problème vient peut être de là

ressource: https://fr.classic.wowhead.com/quest=4450/grand-livre-de-tanaris

**Comment by Ferro3** [comment]
Rectification, le livre est bien lootable, mais seulement si on a fait la quête de la clé [46] [La Clé de la liberté] https://woehead.way-of-elendil.fr/?quest=4451
Du coup y a peut être un problème de filtre de visibilité de la quête [45] [Piégé !] https://woehead.way-of-elendil.fr/?quest=4449
ou bien ça a été fix

ressource wowhead : [45] [Piégé !] https://fr.classic.wowhead.com/quest=4449/pi%C3%A9g%C3%A9

**Comment by Amilgaull** [comment]
A aucun endroit il est indiqué qu'il faut avoir fait cela pour pouvoir looter l'item. Je viens d'ailleurs de tester en ayant fait les quêtes précédentes et le livre reste non lootable.
https://fr.wowhead.com/item=11727/livre-de-bonnacier#english-comments

**Comment by Ferro3** [comment]
> A aucun endroit il est indiqué qu'il faut avoir fait cela pour pouvoir looter l'item
j'ai la même conclusion
> le livre reste non lootable.
curieux, comme je l'ai précisé juste au dessus, j'ai réussi à le looter le mois dernier ; du coup, je comprends plus rien

il y a une option en parlant aux toilettes pour demander à glisser le livre en dessous, tu l'as testé ?

**Comment by Amilgaull** [comment]
Oui, j'ai tenté du coup sans la quête, puis j'ai retenté en ayant validé la quête que tu avais dis mais sans succès et ce bien évidemment en parlant aux toilettes pour qu'il jette le livre à terre^^

**Comment by Ferro3** [comment]
Testé ce jour sur PTR, tout est fonctionnel.
Le livre est lootable, la bulle est maintenant présente, je clôture.
